src/main/scala/com/crealytics/spark/v2/excel/ExcelParser.scala

```diff
@@ -247,7 +254,7 @@ class ExcelParser(dataSchema: StructType, requiredSchema: StructType, val option
               datum.getCellType match {
                 case CellType.NUMERIC | CellType.FORMULA => {
                   /* Excel to spark precision */
-                  datum.getNumericCellValue * 1000
```

Comment:
Why is this not needed anymore? Does Spark do something differently if it gets a Long value?

Reply:
Hi Martin,

It was incorrect before. This code section gets called when a cell is not formatted as Date or Time, however, users pass a custom schema and want to load that cell value as Timestamp, so there is no binding logic to multiply it to 1000.

We haven't seen a case like this in real life yet. And only when handling DateType, I recognized this issue.